core-web/apps/dotcms-ui/.eslintrc.json

```diff
@@ -25,12 +24,13 @@
                         "prefix": "dot",
                         "style": "kebab-case"
                     }
-                ]
+                ],
+                "@typescript-eslint/no-duplicate-enum-values": "off"
```

Comment:
Why?

Reply:
Because in a string enum if you have a value duplicated the lint jumps up.
```
enum NOTIFY {
    NAME = 0,
    NAME2 = 0
}
```